embark/uploader/views.py

```diff
@@ -137,6 +125,9 @@ def start_analysis(request, refreshed):
             logger.error(form.errors)
             return HttpResponse("Invalid Form")
 
+    FirmwareForm.base_fields['firmware'] = forms.ModelChoiceField(queryset=FirmwareFile.objects)
+    DeleteFirmwareForm.base_fields['firmware'] = forms.ModelChoiceField(queryset=FirmwareFile.objects)
```

Comment:
This is not needed anymore. Its in forms.py

Reply:
Do you want me to delete this?

Reply:
I deleted them myself, just for clarification.